bitnami/influxdb/README.md

```diff
@@ -0,0 +1,348 @@
+| `influxdb.extraEnvVars`                     | Array containing extra env vars to configure InfluxDB                                                                                                                                     | `nil`                                                   |
+| `influxdb.replicaCount`                     | The number of InfluxDB replicas to deploy                                                                                                                                                 | `1`                                                     |
+| `influxdb.antiAffinity`                     | Pod anti-affinity policy                                                                                                                                                                  | `soft`                                                  |
+| `influxdb.nodeAffinity`                     | Node affinity policy                                                                                                                                                                      | `{}` (The value is evaluated as a template)             |
+| `influxdb.nodeSelector`                     | Node labels for pod assignment                                                                                                                                                            | `{}` (The value is evaluated as a template)             |
+| `influxdb.tolerations`                      | Tolerations for pod assignment                                                                                                                                                            | `[]` (The value is evaluated as a template)             |
+| `influxdb.securityContext.enabled`          | Enable security context for InfluxDB                                                                                                                                                      | `true`                                                  |
+| `influxdb.securityContext.fsGroup`          | Group ID for the InfluxDB filesystem                                                                                                                                                      | `1001`                                                  |
+| `influxdb.securityContext.runAsUser`        | User ID for the InfluxDB container                                                                                                                                                        | `1001`                                                  |
+| `influxdb.resources`                        | The [resources] to allocate for container                                                                                                                                                 | `{}`                                                    |
```

Comment:
Are we missing a link in [resources]?

Reply:
Nope, I just copied it from other chart that uses that.